src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import "./bulma.css";
```

Comment:
You have two different formats here for importing a css file : do you want to use single quotes or double quotes. 
It is a good practice to choose one way of doing things and replicating it every where or else you will have different results errors that pop up sometimes. 
You can use this to easily format your code before committing https://prettier.io/